py3status/modules/gpmdp.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+Display currently playing song from Google Play Music Desktop Player.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds (5s default)
+
+Requires
+    - gpmdp
+    - gpmdp-remote
+
+@author Aaron Fields https://twitter.com/spirotot
+@license BSD
+"""
+
+# import your useful libs here
+from time import time
+from subprocess import check_output
+import shlex
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+
+    cache_timeout = 5
+
+    def gpmdp(self, i3s_output_list, i3s_config):
+        command = 'gpmdp-remote info'
+        result = u'♫ ' + check_output(shlex.split(command)).decode('ascii', 'ignore').strip()
```

Comment:
Here it might be nice to allow the user to specify a format  eg `♫ {title}` like other modules do.
eg https://github.com/ultrabug/py3status/blob/master/py3status/modules/external_script.py